### ISSUE TITLE
Nav element should not have role="toolbar"

### ISSUE THREAD

**Issue by will0684** [gcdevops/HRWhiteListing#187]
This fails WCAG best practice of using correct aria roles on appropriate elements. You can see the element and selector here:

<img width="640" alt="Screen Shot 2020-04-14 at 12 54 48 PM" src="https://user-images.githubusercontent.com/31868510/79252223-45653c00-7e4f-11ea-8613-b7768ac02ba0.png">

You can read more here:

https://dequeuniversity.com/rules/axe/3.5/aria-allowed-role
